pkgs/tools/security/yubihsm-shell/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, openssl
+, libusb
```

Comment:
error: Function called without required argument "libusb" at /home/jtojnar/Projects/nixpkgs/pkgs/tools/security/yubihsm-shell/default.nix:6, did you mean "librsb", "libusb1" or "ibus"?